tools/gbench/report.py

```diff
@@ -66,6 +68,22 @@ def calculate_change(old_val, new_val):
     return float(new_val - old_val) / abs(old_val)
 
 
+def filter_benchmark(json_orig, family, replacement=""):
+    """
+    Apply a filter to the json, and only leave the 'family' of benchmarks.
+    """
+    regex = re.compile(family)
+    filtered = {}
+    filtered['benchmarks'] = []
+    for be in json_orig['benchmarks']:
+        if not regex.search(be['name']):
+            continue
+        filteredbench = copy.deepcopy(be) # Do NOT modify the old name!
+        filteredbench['name'] = regex.sub(replacement, filteredbench['name'])
```

Comment:
i think this is where we're ending up with `/8` etc. i wonder if it would look better as `BM_copy|BM_memcpy/8` or something similar. this would also help if the report is output somewhere later.

Reply:
> i think this is where we're ending up with `/8` etc.

Yes.

> i wonder if it would look better as `BM_copy|BM_memcpy/8` or something similar.

It will become rather convoluted with larger, more complex names.

> this would also help if the report is output somewhere later.

Yes, _something_ better than now would be nice.